packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -229,4 +229,145 @@ void main() {
 
     expect(find.text('Subtitle'), findsOneWidget);
   });
+
+  testWidgets('ExpansionTile custom tween colors', (WidgetTester tester) async {
+    final Key expandedTitleKey = UniqueKey();
+    final Key collapsedTitleKey = UniqueKey();
+    final Key expandedIconKey = UniqueKey();
+    final Key collapsedIconKey = UniqueKey();
+    const Color expandedTitleColor = Colors.blue;
+    const Color collapsedTitleColor = Colors.red;
+    const Color expandedIconColor = Colors.green;
+    const Color collapsedIconColor = Colors.orange;
+    final ColorTween headerColorTween = ColorTween(
+      begin: collapsedTitleColor,
+      end: expandedTitleColor,
+    );
+    final ColorTween iconColorTween = ColorTween(
+      begin: collapsedIconColor,
+      end: expandedIconColor,
+    );
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: SingleChildScrollView(
+            child: Column(
+              children: <Widget>[
+                const ListTile(title: Text('Top')),
+                ExpansionTile(
+                  initiallyExpanded: true,
+                  headerColorTween: headerColorTween,
+                  iconColorTween: iconColorTween,
+                  title: TestText('Expanded', key: expandedTitleKey),
+                  children: const <Widget>[ListTile(title: Text('0'))],
+                  trailing: TestIcon(key: expandedIconKey),
+                ),
+                ExpansionTile(
+                  initiallyExpanded: false,
+                  headerColorTween: headerColorTween,
+                  iconColorTween: iconColorTween,
+                  title: TestText('Collapsed', key: collapsedTitleKey),
+                  children: const <Widget>[ListTile(title: Text('0'))],
+                  trailing: TestIcon(key: collapsedIconKey),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+
+    Color iconColor(Key key) => tester.state<TestIconState>(find.byKey(key)).iconTheme.color;
+    Color textColor(Key key) => tester.state<TestTextState>(find.byKey(key)).textStyle.color;
+
+    expect(textColor(expandedTitleKey), expandedTitleColor);
```

Comment:
Maybe compare with the expected color like:
```dart
    expect(textColor(expandedTitleKey), Colors.blue);
```

I find this easier to follow since my immediate assumption was that this test was simply checking to see if its color was equal to itself.